tests/load/pipeline/test_stage_loading.py

```diff
@@ -203,9 +204,13 @@ def test_all_data_types(destination_config: DestinationTestConfiguration) -> Non
         exclude_types=exclude_types, exclude_columns=exclude_columns
     )
 
-    # bigquery cannot load into JSON fields from parquet
+    # clickhouse json is experimental, will not work for parquet and makes strange changes for jsonl
```

Comment:
there is another strange issue in the json object in clickhouse, where a nested list that has [1,2,3,"4"] as a value end's up being a list of strings in the destination. another reason to maybe disabled json by default for now.

Reply:
let's disable by default!